src/components/events/eventListingCard.js

```diff
@@ -51,31 +77,49 @@ const CardPrice = styled.div`
   font-weight: 600;
   padding: 5px 10px;
   border-radius: 5px;
-  font-size: 1rem;
+  font-size: 0.75rem;
+
+  ${media.tablet`
+    font-size: 1rem;
+  `};
 `
 
 const CardHeading = styled.h2`
   margin-bottom: 0;
-  line-height: 1.21;
+  line-height: 1.25;
+  font-size: 1rem;
   color: black;
+
+  ${media.tablet`
+    line-height: 1.21;
+    font-size: 1.5rem;
+  `};
 `
 
 export const EventListingCard = props => {
   const { event } = props
 
   return (
     <Card to={`/events/${event.id}`}>
-      <CardImage
+      <CardImageWrapper
         src={`${event.eventsListPicture.file.url}?fit=fill&w=400&h=225&f=faces`}
-        alt={event.eventsListPicture.title}
-        width="400"
-        height="225"
-      />
+      >
+        <CardImage
+          src={`${
+            event.eventsListPicture.file.url
+          }?fit=fill&w=400&h=225&f=faces`}
+          alt={event.eventsListPicture.title}
+          width="400"
+          height="225"
+        />
+      </CardImageWrapper>
       <CardBody>
         <CardDate>{formatDate(event)}</CardDate>
         <CardHeading>{event.name}</CardHeading>
       </CardBody>
-      <CardPrice>from £{event.eventPriceLow}</CardPrice>
+      <CardPrice>
+        {event.isFree ? 'Free' : `from £${event.eventPriceLow}`}
```

Comment:
the `isFree` field has been deprecated. We should be using `eventPriceLow` and `eventPriceHigh` to determine whether an event is free or not.